pkg/referrerstore/plugin/skel/skel.go

```diff
@@ -69,7 +69,7 @@ func PluginMain(name, version string, listReferrers ListReferrers, getBlobConten
 	}
 }
 
-func (c *pcontext) pluginMainCore(name, version string, listReferrers ListReferrers, getBlobContent GetBlobContent, getRefManifest GetReferenceManifest, getSubDesc GetSubjectDescriptor, supportedVersions []string) *plugin.Error {
+func (c *pcontext) pluginMainCore(_, _ string, listReferrers ListReferrers, getBlobContent GetBlobContent, getRefManifest GetReferenceManifest, getSubDesc GetSubjectDescriptor, supportedVersions []string) *plugin.Error {
```

Comment:
should we just remove the name, and version fields from this method?

Reply:
we can't do this in this specific case. This would constitute a breaking change for all external verifiers.